core/src/main/resources/jenkins/split-plugins.txt

```diff
@@ -34,3 +34,6 @@ trilead-api 2.184 1.0.4
 
 # JENKINS-64107
 sshd 2.281 3.0.1
+
+javax-activation-api 2.329 1.2.0-2
+javax-mail-api 2.329 1.6.2-5
```

Comment:
Weekly release accidentally happened before security release so this will be bumped a version
```suggestion
javax-activation-api 2.330 1.2.0-2
javax-mail-api 2.330 1.6.2-5
```